api/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Multistage dockerfile
+
+####
+# Stage 0:
+#
+# We need a requirements.txt, but we don't need pipenv installed in our final image,
+# so we do a multistage build where we generate the requirements.txt in our first stage,
+# and consume it in our final stage.
+# TODO: This slows down the build somewhat, look at alternatives.
+FROM python:3.6
```

Comment:
is there any particular reason why 3.6 was chosen over later versions?

Reply:
Just because that's what we've got as official version in Openshift - thought I could be wrong, looking at https://github.com/sclorg/s2i-python-container - there is a 3.7 AND 3.8. @DerekRoberts how hard would it be using those in openshift?